integrations/tensorflow/e2e/README.md

```diff
@@ -159,14 +159,14 @@ for each module is as follows:
 
 ```
 /tmp/iree/modules/ModuleName
-├── tf_input.mlir          # MLIR for ModuleName in TF's input dialect
-├── iree_input.mlir        # tf_input.mlir translated to IREE MLIR
-├── backend_name           # e.g. iree_vmla, tf or tf_ref
-│   ├── compiled.vmfb      # flatbuffer of ModuleName compiled to this backend
+├── tf_input.mlir           # MLIR for ModuleName in TF's input dialect
+├── iree_input.mlir         # tf_input.mlir translated to IREE MLIR
+├── backend_name            # e.g. iree_vmla, tf or tf_ref
+│   ├── compiled.vmfb       # flatbuffer of ModuleName compiled to this backend
 │   ├── saved_model
 │   └── traces
-│       ├── trace_function
-│       │   └── log.txt    # A more detailed version of the test logs
+│       ├── trace_function  # Directory storing logs and serialization for each trace.
```

Comment:
drive by: trace_function isn't a very descriptive name, especially when there are two. maybe trace_name_1, trace_name_2, with a comment explaining how the trace names are determined in practice?

Reply:
Done!